seqr/utils/file_utils.py

```diff
@@ -37,10 +37,11 @@ def does_file_exist(file_path, user=None):
     if is_google_bucket_file_path(file_path):
         process = _run_gsutil_command('ls', file_path, user=user)
         success = process.wait() == 0
+        outputs = [line.decode('utf-8').strip() for line in process.stdout]
         if not success:
-            errors = [line.decode('utf-8').strip() for line in process.stdout]
-            logger.info(' '.join(errors), user)
-        return success
+            logger.info(' '.join(outputs), user)
+            return False
+        return outputs
```

Comment:
Change to return the file name or the file name list.
The purpose of the change is to fetch the file list for a path with `*` in it. To get the file list, there are a couple of ways: 1) we could store the file list when merging the file names; 2) we could re-fetch the file list with the `get_gs_file_list` function after the user selection; 3) re-fetch the file list with the `does_file_exist` function. #1 could be complicated because of the need to manage some extra data. For #2, using  `get_gs_file_list` to re-fetch the file list is straightforward. But the function was designed to get a file list under a folder, and no wildcard (*) in the path is supported. The change for #3 is small but may not be straightforward for the function itself.